### DISCUSSION TITLE
On MIMO in real-world  5G RAN

### DISCUSSION THREAD

**Discussion by orklann** [srsran/srsRAN_Project#457] [General Help] [answered]
Can anyone help me understand what MIMO is doing on RAN in real-world case?

Let's assume we configure 66 subcarriers in our real-world 5G RAN, is it right that the best case is to have 66x2 MIMO channels (attenas), so that we can maximize the RAN performance?

**Comment by pgawlowicz** [comment]
I am not sure if correctly understand your question. Could you elaborate a bit more on what you want to achieve?

**Reply by orklann** [reply]
I want a example explanation of what a normal real-world RAN deployment looks like, for example, If a  real-world RAN in a cell has 66 subcarriers configuration, does it mean deploy a 66x2 MIMO attenas has the best performance and capacity. Or in other words, for a 66 subcarriers configuration RAN, what's the best number of MIMO could maximize the performance of that RAN.

**Reply by pgawlowicz** [reply, marked as answer]
not sure what do you mean by 66 subcarrier configuration. 

In 5G NR, the bandwidth is set in number of PRBs, which contains 12 subcarriers. The smallest bandwidth is 5MHz, with 25 PRBs and 300 subcarriers (in the case of 15kHz subcarrier spacing). Please check the details here: http://howltestuffworks.blogspot.com/2019/11/5g-nr-resource-blocks.html

MIMO allows sending multiple data streams at the same time, hence, it allows multiplying throughput by a factor of 2-4x (depending on number of antennas). However, the real performance depends on the wireless channel.

**Reply by orklann** [reply]
Thank you for the correction on my example and the link. I just misunderstood the specs on the number of subcarriers in 5G. And I now get the answer I want.
